clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -17,16 +17,17 @@
 #include "clang/Basic/Version.h"
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/SmallVector.h"
-#include "llvm/TargetParser/Triple.h"
 #include "llvm/ADT/Twine.h"
 #include "llvm/BinaryFormat/ELF.h"
+#include "llvm/Bitcode/BitcodeReader.h"
 #include "llvm/Bitcode/BitcodeWriter.h"
 #include "llvm/IR/Constants.h"
 #include "llvm/IR/GlobalVariable.h"
 #include "llvm/IR/IRBuilder.h"
 #include "llvm/IR/LLVMContext.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/PassManager.h"
+#include "llvm/TargetParser/Triple.h"
```

Comment:
Please do not include formatting changes that are not part of your PR. It might confuse pull-down efforts.

Thanks

Reply:
This change was done so that PR could pass clang-format.

Reply:
It is ok to avoid these changes and let clang-format test fail.
Pulldown pain due to such changes is considerable.

Thanks

Reply:
Header re-ordering has been undone.